### ISSUE TITLE
output attribute meaning

### ISSUE THREAD

**Issue by YL-123** [dangweili/pedestrian-attribute-recognition-pytorch#9]
Hello, thanks for sharing your great work!
Could you help me understand the following sentence in the paper, "the DeepMAR output a label vector which represents whether it has each attribute or not"? For example, in the case of the age prediction, PETA dataset which had four attributes 16-30, 31-45, 45-60 and >60 in your selected 35 attributes, If more than 1 output has a positive score, (eg.  first twos) it means the image has both 16-30 and 31-45 attribute? or we need sort above four output scores and vote the biggest positive number as the age prediction? I noticed one more attribute <16 later in the PETA datasets. have you performed some experiments about this? thanks again!

**Comment by YL-123** [comment]
To my first question, I think to vote the max number as the age prediction is a better way. 
 I am wondering is there a way that I could add some softmax above certain attributes, like group softmax in PyTorch?

**Comment by dangweili** [comment]
@YL-123 I treat the attribute classification as a multi-label classification problem, which is a simple way.  As you said, different attributes may be conflict with each other, e.g. age16-30 and age13-45, thus using softmax on these mutually exclusive attributes would be better where the ambiguity would be decreased.
